backend/application/syson-application/src/test/java/org/eclipse/syson/application/imports/ImportSysMLModelTest.java

```diff
@@ -802,4 +882,37 @@ private void assertOperatorExpressionGuard(TransitionUsage t1, String expectedOp
         assertThat(arguments.get(0)).isInstanceOf(expectedFirstParameterType);
         assertThat(arguments.get(1)).isInstanceOf(expectedSecondParameterType);
     }
+    
+    private boolean isSubSettingFromStandardLib(Feature feature, String expectedSubSettedFeature) {
+        return feature.getOwnedSpecialization().stream()
+            .filter(Subsetting.class::isInstance)
+            .map(Subsetting.class::cast)
+                .anyMatch(f -> f.getSubsettedFeature() != null && expectedSubSettedFeature.equals(f.getSubsettedFeature().getQualifiedName()));
+    }
+
+    private void checkMetadaUsage(MetadataUsage metadata, String expectedDefinitionName, String annotatedElementName, Map<Feature, String> attributeNameValue) {
+        assertThat(metadata.getMetadataDefinition().getName()).isEqualTo(expectedDefinitionName);
+        assertThat(metadata.getAnnotatedElement()).hasSize(1).as("Should annotate the element " + annotatedElementName).allMatch(e -> annotatedElementName.equals(e.getName()));
+
+        EList<Feature> features = metadata.getOwnedFeature();
+        assertThat(features).hasSize(attributeNameValue.size());
+
+        for (Entry<Feature, String> entry : attributeNameValue.entrySet()) {
+            Optional<ReferenceUsage> matchingFeature = features.stream()
+                    .filter(ReferenceUsage.class::isInstance)
+                    .map(ReferenceUsage.class::cast)
+                    .filter(ref -> ref.redefines(entry.getKey()))
+                    .findFirst();
+            assertThat(matchingFeature).get()
```

Comment:
those two lines should probably be one

Reply:
I'm not sure I understand what you mean.

Is it?

```java
for (Entry<Feature, String> entry : attributeNameValue.entrySet()) {
            assertThat(features.stream()
                    .filter(ReferenceUsage.class::isInstance)
                    .map(ReferenceUsage.class::cast)
                    .filter(ref -> ref.redefines(entry.getKey()))
                    .findFirst()).get()
                            .satisfies(f -> this.assertStringValue(f, entry.getValue()));
        }
```

I find less readable?

Reply:
Done.